bin/latest-stable

```diff
@@ -14,16 +14,20 @@ if [ -n "${GITHUB_API_TOKEN:-}" ]; then
 	curl_opts=("${curl_opts[@]}" -H "Authorization: token $GITHUB_API_TOKEN")
 fi
 
-# curl of REPO/releases/latest is expected to be a 302 to another URL
-# when no releases redirect_url="REPO/releases"
-# when there are releases redirect_url="REPO/releases/tag/v<VERSION>"
-redirect_url=$(curl "${curl_opts[@]}" "$GH_REPO/releases/latest" | sed -n -e "s|^location: *||p" | sed -n -e "s|\r||p")
-version=
-printf "redirect url: %s\n" "$redirect_url" >&2
-if [[ "$redirect_url" == "$GH_REPO/releases" ]]; then
-	version="$(list_all_versions | sort_versions | tail -n1 | xargs echo)"
+if grep -q "nightly" <<<"$1"; then
+	version="$(get_latest_nightly)"
 else
-	version="$(printf "%s\n" "$redirect_url" | sed 's|.*/tag/v\{0,1\}||')"
+  # curl of REPO/releases/latest is expected to be a 302 to another URL
+  # when no releases redirect_url="REPO/releases"
+  # when there are releases redirect_url="REPO/releases/tag/v<VERSION>"
+  redirect_url=$(curl "${curl_opts[@]}" "$GH_REPO/releases/latest" | sed -n -e "s|^location: *||p" | sed -n -e "s|\r||p")
```

Comment:
What in case when curl's request return an error?

Reply:
Is it possible to make it more informative in such case? Don't want to be too picky, but having empty strings in case of error doesn't seem to be the best approach. Having
```
redirect_url:
version:
```
printed out seems a poor approach.